dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
```

Comment:
Strictly speaking this only ensures that the interstitial has been rendered on the server, not that the user has actually seen it and made a decision to either dismiss or respond to it. This seemed like the most straightforward way to handle this, but if it seems like it might be a problem, we could also wait to set the race_interstitial_shown flag until the dialog is actually dismissed, it's just a little more complicated.